src/plugins/auctionHouseModule/createBid.ts

```diff
@@ -17,12 +47,30 @@ export type CreateBidOperation = Operation<
 >;
 
 export type CreateBidInput = {
+  auctionHouse: AuctionHouse;
+  wallet?: PublicKey | Signer; // Default: identity
```

Comment:
Since we're adding a `seller` parameter here, let's call this buyer so the two are complementary. Could we do the same with the create listing operation for consistency? (there, `wallet` would be `seller`).

```suggestion
  buyer?: PublicKey | Signer; // Default: identity
```